paddlenlp/transformers/bert/modeling.py

```diff
@@ -397,6 +418,73 @@ def forward(self,
                 position_ids=None,
                 attention_mask=None,
                 output_hidden_states=False):
+        r'''
+        The BertModel forward method, overrides the `__call__()` special method.
+
+        Args:
+            input_ids (Tensor):
+                Indices of input sequence tokens in the vocabulary. They are
+                numerical representations of tokens that build the input sequence.
+                Its data type should be `int64` and it has a shape of [batch_size, sequence_length].
+            token_type_ids (Tensor, optional):
+                Segment token indices to indicate first and second portions of the inputs.
+                Indices can either be 0 or 1:
+
+                - 0 corresponds to a *sentence A* token,
+                - 1 corresponds to a *sentence B* token.
+
+                Its data type should be `int64` and it has a shape of [batch_size, sequence_length].
+                Defaults to `None`, which means we don't add segment embeddings.
+            position_ids(Tensor, optional):
+                Indices of positions of each input sequence tokens in the position embeddings. Selected in the range ``[0,
+                max_position_embeddings - 1]``.
+                Defaults to `None`. Shape as `(batch_size, num_tokens)` and dtype as `int32` or `int64`.
+            attention_mask (Tensor, optional):
+                Mask to avoid performing attention on padding token indices.
+                The values should be either 0 or 1.
+
+                - **1** for tokens that **not masked**,
+                - **0** for tokens that **masked**.
+
+                It's data type should be float32 and has a shape of [batch_size, sequence_length].
+                Defaults to `None`.
+            output_hidden_states (bool, optional):
+                Whether to return the output of each layer in the hidden layers.
```

Comment:
Whether to return the output of each hidden layers.

Reply:
done